docs/fud/axi-gen.md

```diff
@@ -0,0 +1,72 @@
+# Axi generation
+
+Calyx currently generates a fairly complex AXI interface that can be daunting
+to deal with if confronting for the first time.
+The following is an overview of how the generation occurs
+and how the output AXI interface behaves as of 2022-9-11.
+
+In general, when `fud` is asked to create an [`.xclbin` file][xclbin] a `kernel.xml`,
+`main.sv`, and `toplevel.v` are created as intermediate steps for [xilinx tools][xilinx_tools]
+to properly work.
+
+`main.sv` contains the SystemVerilog needed for our computations to perform
```

Comment:
We should mention that the file is compiled with slightly different flags, specifically using the `--synthesis` and `-p external` pass pipeline